recipe/meta.yaml

```diff
@@ -24,9 +24,19 @@ source:
   sha256: 88f29bec81880844da2eee13f1cdc51e1ccd8b4c60af6df633577810b30f80d0  # [win]
 
 build:
-  number: 0
+  number: 1
   skip: true  # [osx]
 
+test:
+  commands:
+    - test -L $PREFIX/lib/libcublas.so.{{ version }}                                          # [linux]
```

Comment:
So just to confirm, moving these `test`s to the top-level worked?

If so, would move these to after `outputs`, but before `about` in the top-level portion of the recipe

Reply:
Yes, moving this to the top level worked. I verified in the CI output.

The conda-forge linter won't let us put this key in any other place:

> For **recipe**:
> 
>     * The top level meta keys are in an unexpected order. Expecting ['package', 'source', 'build', 'test', 'outputs', 'about', 'extra'].